tests/test_integration.py

```diff
@@ -183,11 +181,15 @@ def do_request(self, req_num):
         stripe.api_base = "http://localhost:%s" % self.mock_server_port
         stripe.enable_telemetry = True
 
-        stripe.Balance.retrieve()
-        stripe.Balance.retrieve()
+        cus = stripe.Customer("cus_xyz")
+        cus.description = "hello"
+        cus.save()
+
+        stripe.Customer.retrieve("cus_xyz")
+        stripe.Customer.retrieve("cus_xyz")
+
+        reqs = MockServerRequestHandler.get_requests(3)
 
```

Comment:
Bring back this assertion?
```suggestion
        assert MockServerRequestHandler.num_requests == 3
        
```

Reply:
It's actually covered by `reqs = MockServerRequestHandler.get_requests(3)` -- this is cleanup I forgot to do when I introduced the test server here: https://github.com/stripe/stripe-python/pull/1145/files#diff-98bfe78f0e8d895dd454b5eef6f6485a80526776ad3b6525a14117029bd1eacbR39